lacework-agent/values.yaml

```diff
@@ -87,6 +97,18 @@ laceworkConfig:
   # [Optional] Kubernetes cluster name
   # https://support.lacework.com/hc/en-us/articles/360005263034-Deploy-on-Kubernetes
   kubernetesCluster:
+  # [Optional] Define the behavior of the agent's package scans
+  packagescan:
```

Comment:
Why did `packagescan` appear twice in this file?
Seems like a bug to me.

Reply:
That's correct @c-p-i-o. @brooks-connor-a can you check in the changes from 243a640473dff4ae9d4ffd7b9abc29b675bf7770 and 243a640473dff4ae9d4ffd7b9abc29b675bf7770 in the agent repo?

Reply:
@nschmeller, that's the same commit hash twice. I do see the fix that went in two days ago, in main. I'll update this PR with the fix, and let me know if theres a second change needed as well

Reply:
Yes, sorry, was on mobile :) that's the one, minus the properties test change

Reply:
✅